test/axes/axisTests.ts

```diff
@@ -655,4 +655,58 @@ describe("Axis", () => {
     axis.destroy();
     div.remove();
   });
+
+  describe("tickLabelDataOnElement", () => {
+    let div: d3.Selection<HTMLDivElement, void | {}, any, any>;
+    let scale: Plottable.Scales.Category;
+    let axis: Plottable.Axes.Category;
+
+    function setup(domain: string[]) {
```

Comment:
you can probably put this in a `beforeEach()` by having just one domain of ["label1", "label2", "long..."]